backend/sse/src/server.ts

```
@@ -9,42 +9,66 @@
+};
+
+// destructuring the service definitions
+const [hiddenTestProcessProto, referenceSolution] = loadProtos(PROTO_PATHS);
+
+const hiddenTestProcessService = hiddenTestProcessProto.hidden_test_process.HiddenTestProcess.service;
+const referenceSolutionService = referenceSolution.reference_solution.ReferenceSolution.service;
+
+/******************** Redis Connection ********************/
 const redis = new Redis({
     host: process.env.REDIS_HOST,
     port: parseInt(process.env.REDIS_PORT || "6379"),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)